mep/common/tests.py

```diff
@@ -527,3 +531,83 @@ class MyViewWithFacets(FacetJSONMixin):
         assert isinstance(response, JsonResponse)
         assert response.content == b'{"facets": "foo"}'
 
+
+class TestRdfViewMixin(TestCase):
+
+    def test_get_uri(self):
+        class MyRdfView(RdfViewMixin):
+            pass
+        view = MyRdfView()
+        # get_uri() not implemented by default
+        with pytest.raises(NotImplementedError):
+            view.get_uri()
+
+    def test_get_breadcrumbs(self):
+        class MyRdfView(RdfViewMixin):
+            breadcrumbs = [('Home', '/'), ('My Page', '/my-page')]
+            def get_uri(self):
+                return ''
+        view = MyRdfView()
+        # should add provided breadcrumbs to context
+        context = view.get_context_data()
+        assert context['breadcrumbs'] == [('Home', '/'), ('My Page', '/my-page')]
+
+    def test_rdf_graph(self):
+        class MyRdfView(RdfViewMixin):
+            breadcrumbs = [('Home', '/'), ('My Page', '/my-page')]
+            def get_uri(self):
+                return 'http://jimcasey.lifestyle/my-page'
+        view = MyRdfView()
+        graph = view.as_rdf()
+        page_node = rdflib.URIRef('http://jimcasey.lifestyle/my-page')
+        # should have root node as URIRef to the page with default type WebPage
+        assert (page_node,
+                rdflib.RDF.type,
+                SCHEMA_ORG.WebPage) in graph
+        # should have (blank) nodes with URIs for each of the breadcrumbs
+        # any=False will raise an exception unless we find exactly one match
+        home_crumb = graph.value(predicate=SCHEMA_ORG.item,
+                                 object=rdflib.Literal('/'),
+                                 any=False)
+        page_crumb = graph.value(predicate=SCHEMA_ORG.item,
+                                 object=rdflib.Literal('/my-page'),
+                                 any=False)
+        # crumbs should have the correct name and position values
+        assert graph.value(home_crumb, SCHEMA_ORG.name) == rdflib.Literal('Home')
+        assert graph.value(page_crumb, SCHEMA_ORG.name) == rdflib.Literal('My Page')
+        assert graph.value(home_crumb, SCHEMA_ORG.position) == rdflib.Literal(1)
+        assert graph.value(page_crumb, SCHEMA_ORG.position) == rdflib.Literal(2)
+        # page should have breadcrumb list
+        crumb_list = graph.value(page_node, SCHEMA_ORG.breadcrumb, any=False)
+        # crumbs should belong to the list as itemListElements
+        assert (crumb_list, SCHEMA_ORG.itemListElement, home_crumb) in graph
+        assert (crumb_list, SCHEMA_ORG.itemListElement, page_crumb) in graph
+
+
+class TestBreadcrumbsTemplate(TestCase):
```

Comment:
This is nice, great way to test the template directly.  You could get the template in a `setUp` method rather than each test.